nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_kernel.py

```diff
@@ -511,7 +511,7 @@ def build(self, target_model: NNCFNetwork) -> ElasticKernelHandler:
                 # Padding
                 ap = ElasticKernelPaddingAdjustment(elastic_kernel_op)
                 pad_op = UpdatePadding(ap).to(device)
-                nncf_logger.warning('Padding will be adjusted for {}'.format(node_name))
+                nncf_logger.warning(f'Padding will be adjusted for {node_name}')
```

Comment:
Propose moving to debug
@jpablomch

Reply:
@ljaljushkin Moving to debug works. Thanks!

Reply:
Done